registry.yaml

```diff
@@ -398,10 +398,10 @@ packages:
   replacements:
     386: i386
     amd64: x86_64
```

Comment:
```yaml
rosetta2: true
```

is required.

Reply:
OK. I noticed that there is no release for `darwin_arm64` without latest it.

Reply:
Oh, sorry. I overlooked the darwin_arm64 of the latest version.
I'll fix it using `version_overrides`.
https://aquaproj.github.io/docs/reference/registry-config#version_constraint-version_overrides